src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Apart from most likely being larger and differ from other vars we define (so it would be harder to comprehend) it would require people to write sth like `timers.setTimeout` instead of `setTimeout`.

So it might be larger and it would be harder to use. I see no gain in grouping those modules into one, only drawbacks.